src/technologies/p.json

```
@@ -3394,6 +3394,18 @@
     "implies": "Java",
     "website": "https://www.publiccms.com"
   },
+  "PubTech": {
+    "cats": [
+      67
+    ],
+    "description": "PubTech is a consent management platform helping brands and businesses collect, store and leverage their customer consents.",
+    "icon": "PubTech.png",
+    "scriptSrc": [
```

Looks like we can also match on this global variable
```suggestion
    "js": {
      "__pub_tech_cmp_config": ""
    },
    "scriptSrc": [
```

---

Great suggestion, let's add it